src/index.html

```diff
@@ -4,11 +4,226 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+<style>
+
+h1 {
+font-size: 80px; 
+text-align: center;
+}
+
+h2 {
+  text-align: left;
+}
+
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+img {
+  opacity: 1.0;
+  filter: alpha(opacity=100); /* For IE8 and earlier */
+}
+
+img:hover {
+  opacity: 0.5;
+  filter: alpha(opacity=60); /* For IE8 and earlier */
+}
+.row .col {
+  float: left;
+  width: 13%;
+  border: 10px solid #fff;
+}
+
+
+table,td,th
+  {
+  border:0px solid black;
+  }
+
+table
+  {
+  width:100%;
+  }
+
+th
+  {
+  height:50px;
+  }
+td
+{
+text-align:left;
+}
+label{float: left;width: 80px;line-height:25px;} 
+form{margin:0px} 
+input{width: 180px;border:1px solid #808080;} 
+textarea{width: 250px;height: 150px;} 
+.main{
+ text-align: center;
+border-radius: 20px;
+width: 1500px;
+height: 350px;
+margin: auto;
+position: absolute;
+  top: 0;
+left: 0;
+ right: 0;
+ bottom: 0;
+}
+</style>
   </head>
   <body>
-    <div id="main"></div>
+    <div class="main">
+    
     <script src="./js/index.js"></script>
```

Comment:
Javascript should be included at the end of the document.
The reason is JS should be executed after the HTML DOM is loaded.

I think your point is you might use the  document.ready from jQuery, but I think we might not require jQuery for this homework.